broker/unified_sql/src/stream_sql.cc

```diff
@@ -2858,8 +2921,44 @@ void stream::_process_tag(const std::shared_ptr<io::data>& d) {
   if (tg.action() != Tag_Action_DELETE) {
     *st << *s;
     int32_t conn = special_conn::tag % _mysql.connections_count();
-    _mysql.run_statement(*st, database::mysql_error::store_tag, false, conn);
+    std::promise<uint64_t> p;
```

Comment:
Because of last modifications in the DB, this way to update the db won't work. Please look at my changes in the severity code and you should be able to adapt this code.